packages/neuron-wallet/src/services/node.ts

```diff
@@ -187,6 +200,50 @@ class NodeService {
         return false
       })
   }
+
+  private getInternalNodeVersion() {
+    const appPath = electronApp.isPackaged ? electronApp.getAppPath() : path.join(__dirname, '../../../..')
+    const ckbVersionPath = path.join(appPath, '.ckb-version')
+    if (fs.existsSync(ckbVersionPath)) {
+      try {
+        return fs.readFileSync(ckbVersionPath, 'utf8')?.split('\n')[0]
+      } catch (err) {
+        logger.error('App\t: get ckb node version failed')
+      }
+    }
+  }
+
+  private async verifyNodeVersion() {
+    const network = NetworksService.getInstance().getCurrent()
+    const localNodeInfo = await new RpcService(network.remote).getLocalNodeInfo()
+    const internalNodeVersion = this.getInternalNodeVersion()
+    if (internalNodeVersion && !localNodeInfo.version?.startsWith(internalNodeVersion.slice(1))) {
```

Comment:
It requires the exact same version. Maybe the version range could be accepted.

Reply:
It's just a notice, maybe not affecting the Neuron running.

Reply:
CKB node follows the semantic versioning so `major.minor` could be treated as the same one.

Usually `major` could be regarded as the same one because only `major` could have breaking changes, but now `major` is `0`, `minor` should be considers. But `patch` shouldn't break the workflow